cloudfoundry.md

```diff
@@ -130,6 +130,8 @@ cf set-env foia DATABASE_URL [value]
 cf set-env foia FOIA_ANALYTICS_ID [value]
 cf set-env foia FOIA_SECRET_SESSION_KEY [value]
 cf set-env foia DJANGO_SETTINGS_MODULE foia_hub.settings.dev
+cf set-env foia NEW_RELIC_LICENSE_KEY=[value]
+cf set-env foia NEW_RELIC_APP_NAME=[value]
```

Comment:
`foia, foia-a, foia-b` are all name of apps in our orgs `foia` space. I'm pretty sure that @konklone created them and set the env variables for them. They should stay constant.